api/pkg/shared/resource/resource.go

```diff
@@ -66,6 +67,7 @@ func (r *Request) Query() ([]model.Resource, error) {
 		filterByTags(r.Tags),
 		filterByKinds(r.Kinds),
 		filterByCatalogs(r.Catalogs),
+		filterByCategories(r.Categories),
```

Comment:
`// Fields: name, []kinds,[]Catalogs []Categories []Tags, Match, Limit`

Reply:
Should this be added as a comment ??

Reply:
comment is the just update with ` []Categories `

Reply:
line no 51